tests/test_models/test_inference/test_forecast.py

```diff
@@ -1115,6 +1151,22 @@ def test_forecast_out_sample_suffix_failed_deepar(self, model, transforms, datas
         with pytest.raises(AssertionError):
             self._test_forecast_out_sample_suffix(ts, model, transforms)
 
+    @pytest.mark.parametrize(
+        "model, transforms, dataset_name",
+        [
+            (
+                TFTNativeModel(encoder_length=7, decoder_length=7, trainer_params=dict(max_epochs=1)),
+                [],
+                "example_tsds",
+            ),
+        ],
+    )
+    def test_forecast_out_sample_suffix_failed_tft(self, model, transforms, dataset_name, request):
+        """This test is expected to fail due to dependence of future predictions on previous predictions in TFT."""
```

Comment:
Should it actually fail? TFT is "direct" model -- future predictions don't depend on previous predictions

Reply:
May be it's not clearly written. It is not autoregressive, but in these cases model uses different contexts